### DISCUSSION TITLE
Trying to upload crashed...

### DISCUSSION THREAD

**Discussion by martykazmaier** [mkrueger/icy_term#35] [General]
First, I couldn't find the file that was definitely in the directly (ironically, icyterm...) so I pasted the filename in.  Then, icyterm abended with no error.  I think I  might actually like the way netrunner supports upload file queues...  It's a lot easier to drag a file into the window than maneuver through the directory tree.  Ok, I just tested it again and the actual directory jumped all over the place, even selecting each directory then "Open".  It was a mess.  Everything but file transfers is working seamlessly and flawlessly!

**Comment by mkrueger** [comment]
Drag & Drop for file uploads is a great idea. I'll look what can be done there.

Y file transfers is the area I paid a bit too less attention atm :/
